gooey/application.py

```diff
@@ -0,0 +1,18 @@
+'''The command line interface.'''
+
+import sys
+from importlib import import_module
+
+from gooey import Gooey
+
+
+def main(args=None):
+    args = args or sys.argv[1:]
+    module_path, function_name = args[0].split(":")
+    if len(args) > 1:
+        if args[1] == "--":
+            del args[1]
+    sys.argv = ["gooey", *args[1:]]
```

Comment:
Also, not sure if it makes sense to accept additional arguments on the command line? Will they be populated in the Gooey window? If not, I'll remove this ability.

Reply:
If you enable the `use_cmd_args` option, it'll pass whatever is supplied on the command line into the GUI. So, if you change your call to something like `Gooey(use_cmd_args=True)(function)` ist (should) read and process any additional args.